### ISSUE TITLE
Random genre tags in list after update

### ISSUE THREAD

**Issue by rmsthebest** [gnumdk/lollypop#1259]
Random genres popped up after update from debian/ubuntu repository.
Afro-beat, Anime, and MPB to name a few. None of my songs have these tags, they are quite useless.

If there's some way to remove them I apologize for not finding that feature, otherwise I'd say something might be wrong.

**Comment by gnumdk** [comment]
Open files with kid3-qt, I'm sure you will find tag frames with this genres ;)

If not, can you send me a file by email?

**Comment by rmsthebest** [comment]
I used quod libet to search the same music for those genres and they also came up empty. 

The only thing I can think of that might have caused this is there used to be some chart feature pulling music from youtube, I activated it once to see what it did, but i never listened to music from it. I'm guessing there were remnants from there that somehow caused the empty genres.

I ended up resetting the database and now they're gone. Still a bit weird but at least it's all good now.
